1-getting-started-lessons/2-github-basics/README.md

```diff
@@ -120,14 +120,24 @@ Let's say you have a folder locally with some code project and you want to start
 
    This command helps us to unstage only a particular file at once that we don't want to include for the next commit.
 
-1. **Persisting your work**. At this point you've added the files to a so called _staging area_. A place where Git is tracking your files. To make the change permanent you need to _commit_ the files. To do so you create a _commit_ with the `git commit` command. A _commit_ represents a saving point in the history of your repo. Type the following to create a _commit_:
+1. **Persisting your work**. At this point you've added the files to a so called _staging area_. A place where Git is tracking
+your files. To make the change permanent you need to _commit_ the files. To do so you create a _commit_ with the `git commit` command. A _commit_ represents a saving point in the history of your repo. Type the following to create a _commit_:
 
    ```bash
    git commit -m "first commit"
    ```
 
    This commits all of your files, adding the message "first commit". For future commit messages you will want to be more descriptive in your description to convey what type of change you've made.
 
+1. **Keep track of commits**. In the real-world applications we will most often deal with mutliple commit. In order to keep track of the commit history git provides `git log` command. 

```

Comment:
`git log` is a great addition to this lesson! There is an issue currently open (#1108) to include this within Task 2 of this lesson. If you'd like, for this PR, this block could be moved to match what is needed for that issue, or remove this block from your PR and it can be added separately.